src/generators/scripts/start.js

```diff
@@ -19,7 +21,10 @@ export default {
   },
 }
 
-function startScriptDocker() {
+function startScriptDocker(config) {
+  if (config.network.txGenerate) {
+    return readFileToString(joinPath(libRootDir(), 'lib', 'start-with-splunk-txns.sh'))
+  }
```

Comment:
Need to make sure we handle each of the possible configs: just Splunk, just Generator, and Splunk+Generator. Or add validation to the tool question that requires specific combinations. I saw that there is also `start-with-txns.sh` but it is not used.

Reply:
working on this. good call.

Reply:
good to go now 👍

Reply:
I think this is much more doable with the changes we've done to simplify the splunk/txgen start scripts. Then we can just get rid of the multiple shell scripts in the lib folder